app/views/courses/show.html.erb

```diff
@@ -2,7 +2,7 @@
 
 <% if permitted_referrer? %>
   <%= content_for(:before_content) do %>
-    <%= govuk_back_link_to(results_path(request.referrer), "Back to search results") %>
+    <%= govuk_back_link_to(request.referrer, "Back to search results") %>
```

Comment:
Is there any chance someone could access this page directly and therefore request.referrer would not be set?

Reply:
Yes, but I think this is the way C# did it so we're replicating that? Can't test this in prod now because we've enabled redirects.

Reply:
@dankmitchell Do you remember offhand?

Reply:
If they access the page directly we don't show the back link. Thats intentional.

Reply:
We should show an alternative link, along the lines of "< Start a search for courses" or something. But this matches what's live for now.

The lack of a back link here never sat well with me.